README.md

```diff
@@ -88,6 +88,8 @@ end
 You'll need a Braintree sandbox account to run the integration tests. Also, be
 sure that your account has [Duplicate Transaction Checking][dtc] disabled.
 
+In order to test the merchant account features, your sandbox account needs to have a master merchant account and it needs to be added to your environment variables(only needed in test).
```

Comment:
Please wrap this at 80 characters and there should be space after "variables", so it says "variables (only needed for testing)."